apps/frontend/vite.config.mts

```diff
@@ -0,0 +1,46 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig, splitVendorChunkPlugin } from 'vite';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [splitVendorChunkPlugin(), react()],
+  server: {
+    open: true,
+    host: true,
+    port: 3000,
```

Comment:
I was just thinking if the port needs to be hardcoded to 3000 or it is better to use an env variable.